scheduler/server/load_based_sched_alg_test.go

```diff
@@ -47,8 +30,13 @@ type testDef struct {
 	classes      map[string]classState
 }
 
+// test the algorithm for various scenarios
+// https://docs.google.com/spreadsheets/d/1QP-okxrVkmKBMtc4OzWe8cFU15u84b9aXENeKqGogbc/edit#gid=554928957 is a
```

Comment:
this can't be seen by people outside twitter's org

Reply:
yes, we shouldn't put these links in open source.